tests/simple-solution-api.spec.ts

```diff
@@ -184,3 +183,127 @@ test('Get order with missing data should receive code 500', async ({ request })
   console.log('response body:', await response.json())
   expect(response.status()).toBe(StatusCodes.INTERNAL_SERVER_ERROR)
 })
+
+// 10 Homework
+//10.1  Scenario to check Low Risk score
+test('post request calculate Low risk score with valid data returns code 200', async ({
+  request,
+}) => {
```

Comment:
better to create a separate class and put loan tests there
this is last comment